homeassistant/components/screenlogic/binary_sensor.py

```diff
@@ -64,17 +65,24 @@ async def async_setup_entry(
 class ScreenLogicBinarySensor(ScreenlogicEntity, BinarySensorEntity):
     """Representation of the basic ScreenLogic binary sensor entity."""
 
+    _attr_has_entity_name = True
+
     @property
     def name(self):
         """Return the sensor name."""
-        return f"{self.gateway_name} {self.sensor['name']}"
+        return self.sensor["name"]
 
     @property
     def device_class(self):
         """Return the device class."""
         device_type = self.sensor.get("device_type")
         return SL_DEVICE_TYPE_TO_HA_DEVICE_CLASS.get(device_type)
 
+    @property
```

Comment:
You can use this as an _attr

Reply:
[77b4529](https://github.com/home-assistant/core/pull/85725/commits/77b4529a8d4895074030adfed3f876bf7b8c725b)